components/mgmtworker/scripts/snapshot_permissions_fixer

```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/env bash
+restservice_user=$(cat /usr/lib/systemd/system/cloudify-restservice.service | grep User | sed 's/User=//')
```

Comment:
But you still need to make sure the cloudify-restservice.serivce file exists before running the scripts.. so you need to have a relationship in order to make the order right (if its not already like that..).